darkmoon/api/v1/metadata/views.py

```diff
@@ -38,7 +28,7 @@ async def list_metadata(
     page: int = Query(0, ge=0, description="The page to iterate to."),
     length: int = Query(10, ge=1, le=500),
 ) -> list[MetadataEntity]:
-    """Return list of metadata that matches the parameters in the database.
+    """List of metadata that matches the parameters in the database.
```

Comment:
This endpoint is not a list, but it grabs a list. You can say something like Get list